### ISSUE TITLE
Instability PK-Sim

### ISSUE THREAD

**Issue by wilbertdew** [Open-Systems-Pharmacology/PK-Sim#1313]
Hi All,

I am working on an expanding PK-Sim project that contains now around 30-40 compounds, the same amount of observed data sets and similations and a similar amount of parameter identifications. The last days, PK-Sim crashes regularly and any unsaved work up to that point is lost. I need to save every quarter or so to minimize loss of work. Does anyone have experience with this or knows the limits of PK-Sim to work normally. I don't get any error messages, just a windows popup that says PK-Sim has stopped working. Using 7.4 build 127. Any help is appreciated!
Best,

Wilbert

**Comment by StephanSchaller** [comment]
@wilbertdew, are your compounds completely different types of compounds or analogs (i.e. possibly most of the ADME properties are similar/the same and only some differ)?
Just a thought: if the latter is the case, you could just create a "representative generic" PK-Sim model for the class of analogs and then move to R to run batches of simulations or PIs by then explicitly specifying the previously generic properties (i.e. parameters) of the compounds.

You can also do this if the compounds are completely different, but then it requires more work in creating the generic PK-Sim model.

Just an idea,
Stephan

**Comment by regulus92** [comment]
Yes, I encounter this problem from time to time also. Maybe my Chinese language environment of Windows matters. What I have done by far is save the work frequently.

**Comment by wilbertdew** [comment]
Thank you all for the help! @msevestre, it was mainly having a few parameter identifications open that seemed to cause the problem. I have 8 GB RAM on my system. I saw in the task manager that this was indeed filling up quickly. The file was 50 MB. I have now separated the file for modalities and species, which seemed to solve the problem, but makes comparisons maybe a bit less easy to perform. Anyway, it works sufficiently smooth right now :)
